style.css

```diff
@@ -1,617 +1,713 @@
+  font-weight: 600;
+  line-height: 16px;
+  color: #7a869a;
 }
 
-.container4 {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 16px;
-  gap: 12px;
-  width: 327px;
-  height: 552px;
-  background: #fff;
-  border: 1px solid #dfe1e6;
-  border-radius: 16px;
-  margin-bottom: 100px;
-  margin-left: 24px;
-  margin-right: 24px;
+.modal--buttons-techs-wrapper ul li {
+  margin-right: 0.8em;
+  padding: 4px 12px;
+  height: 24px;
+  width: auto;
+  background-color: #ebebff;
+  border-radius: 8px;
+  color: #6070ff;
+  font-weight: 500;
+  line-height: 16px;
 }
 
-.placeholder-img4 {
-  background-image: url("./images/4th img.png");
-  background-repeat: no-repeat;
-  background-size: 370px;
-  width: 100%;
-  height: 220px;
+.about-sec > div > ul li {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1em 0.5em;
 }
 
-.text {
-  height: 288px;
-  width: 100%;
+.card-body > ul li:first-child {
+  color: #344563;
 }
 
-.text h3 {
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
+.card-body p {
+  font-family: var(--font-family-primary);
   font-weight: 400;
   font-size: 15px;
   line-height: 24px;
-  margin-top: 20px;
-  margin-left: 20px;
+  color: #344563;
+}
+
+.about-sec {
+  background-color: #fff;
+  padding: 2em 0;
+  margin-top: 5rem;
+}
+
+.about-sec .heading {
+  font-family: var(--font-family-primary);
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
   color: #172b4d;
+  margin-bottom: 0.6em;
 }
 
-.text p {
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
+.about-sec .body {
   font-weight: 400;
-  font-size: 15px;
+  font-size: 16px;
   line-height: 24px;
-  margin-top: 20px;
-  margin-left: 20px;
+  font-family: var(--font-family-primary);
   color: #344563;
 }
 
-.lang-names {
-  display: flex;
-  flex-direction: row;
-}
-
-button {
-  color: #396df2;
-  font-family: "Poppins", sans-serif;
-  font-style: normal;
+.about-sec .languages {
   font-weight: 500;
-  font-size: 17px;
+  font-size: 20px;
   line-height: 24px;
-  background: white;
-  border: 1px solid #396df2;
-  border-radius: 10px;
-  padding: 10px;
-  margin-top: 20px;
-  cursor: pointer;
-}
-
-button:hover {
-  background: #6070ff;
-  box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
-  border-radius: 8px;
-  color: #fff;
+  font-family: var(--font-family-primary);
 }
 
-button:active {
-  transform: translateY(7px);
-  color: #fff;
-  background: #2230d2;
-  border-radius: 8px;
-}
-
-#portfolio {
-  display: grid;
-  grid-template-columns: repeat(1, 1fr);
+.about-sec .social-media-icon {
+  margin: 1em 0;
 }
 
-.about {
+.contact-section {
   background-color: #fff;
 }
 
-.about-sec {
-  margin: 0 1rem;
+.section-wrapper {
+  background: url("./images/header-shapes mobile.png");
+  background-position: right;
+  background-repeat: no-repeat;
+  background-size: contain;
+  padding: 0.5em 0 5em 0;
+  background-color: #6070ff;
+  border-top-left-radius: 5rem;
 }
```

Comment:
Nice job adding the about me section to your project 😄! Please consider the following observations to improve your design 😉:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/206626329-b9c50aba-308e-4c73-902f-03fc51bfb212.png)  | ![image](https://user-images.githubusercontent.com/42799579/206626205-445c4ba7-fbb8-452f-8a8b-69008f09fb34.png)  |

- [ ] Please adjust the margin between the elements "About Myself", the paragraph, and the text "LET'S CONNECT".

- [ ] Kindly make sure that your languages are displayed in a row, instead of a column.

- [ ] Please notice that, in the original design, the languages have darker padding 😉.

By updating this, your project will look closer to the original design! Remember: this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.